resources/devbox/macos/keyman.macos.env.sh

```diff
@@ -20,8 +20,8 @@ if [ -z "$HOMEBREW_PREFIX" ]; then
 fi
 
 export PATH="$HOMEBREW_PREFIX/opt/coreutils/libexec/gnubin:$PATH"
-export PATH="$HOMEBREW_PREFIX/opt/java11/bin:$PATH"
-export JAVA_HOME="$HOMEBREW_PREFIX/opt/java11"
+export PATH="$HOMEBREW_PREFIX/opt/openjdk@21/bin:$PATH"
+export JAVA_HOME="$HOMEBREW_PREFIX/opt/openjdk@21"
```

Comment:
maybe be also good to export `JAVA_HOME_21` to where openjdk 21 is.

The android build.sh will set it again at
https://github.com/keymanapp/keyman/blob/cb7f0ccef6037cfdbb0b9ab901e95eb1ed0155e8/resources/build/build-utils.sh#L333-L344

Reply:
Sounds like it's already handled then?  What's the benefit to exporting it separately from what build-utils.sh is already doing in this excerpt?

Reply:
well, if you need to go between stable-18.0 (on JDK 11) and master (on JDK 21), it might be handy to have the profile export `JAVA_HOME_11` and `JAVA_HOME_21`

Reply:
Note that `stable-18.0` won't have this change; only `master` (19.0) will.  Unless we cherry-pick it, anyway.  It's an in-repo file, after all.